### PR TITLE
FIG-16741(restricted): Remove confidentiality feature

### PR DESCRIPTION
**Pull request by birocorneliu**

### CODE REVIEW

**Review by adrian-castravete — APPROVED** [review]

File: swagger_documentation/swagger_parsed.json [hunk 1]
```diff
@@ -10248,7 +10248,7 @@
                     "type": "string", 
                     "format": "url", 
                     "description": "Article public url", 
-                    "example": "http://figshare.com/articles/media/article_name/2000005"
+                    "example": "http://figshare.com/articles/item_type/article_name/2000005"
```
Comment (adienasoaie) [inline swagger_documentation/swagger_parsed.json:10251]: Sure about this one? Why the change?

**Review by adienasoaie — APPROVED** [review]
